Cargo.toml

```diff
@@ -48,14 +48,14 @@ secrecy = { version = "0.8.0", features = ["alloc", "serde"] }
 serde = "1.0.154"
 serde_json = "1.0.117"
 tracing = "0.1"
-zenoh = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0", features = [
+zenoh = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/admin_keys_remapping", features = [
   "plugins",
   "unstable",
   "internal",
 ] }
 zenoh-plugin-mqtt = { version = "0.11.0-dev", path = "zenoh-plugin-mqtt/", default-features = false }
-zenoh-plugin-rest = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0", default-features = false }
-zenoh-plugin-trait = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0", default-features = false }
+zenoh-plugin-rest = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/admin_keys_remapping", default-features = false }
+zenoh-plugin-trait = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/admin_keys_remapping", default-features = false }
```

Comment:
To be updated after the merge of: https://github.com/eclipse-zenoh/zenoh/pull/1209